### DISCUSSION TITLE
Feature Suggestion: Modify map.fitBounds() to wrap around +-180 degrees longitude

### DISCUSSION THREAD

**Discussion by AdamAtFloreo** [maplibre/maplibre-gl-js#4083] [Ideas]
As an example, let's say I want to call fitBounds([100, -20, -100, 20]) and have it show Asia on the left side, North America on the right. It does not currently do this. Instead it seemingly "corrects" my input to [-100, -20, 100, 20] so that NA is always on the left.

**Comment by HarelM** [comment]
Do different inputs change this behavior? 
I remember there was an issue on this or a discussion...

**Comment by HarelM** [comment]
I think the following is the issue that was somewhat related: 
https://github.com/maplibre/maplibre-gl-js/issues/3048
